Gemfile.lock

```diff
@@ -67,6 +67,16 @@ GIT
     active_model_serializers (0.9.0.alpha1)
       activemodel (>= 3.2)
 
+GIT
+  remote: https://github.com/rshipp/logjam_agent.git
```

Comment:
could we possibly have this as a repo in Braven org?

Reply:
yep, but somebody else has to make it bc I don't have perms

Reply:
made you an owner